client/src/views/DashboardView.vue

```diff
@@ -13,15 +13,16 @@
           <v-card variant="flat" class="pb-1">
             <v-container class="pa-0" fluid>
               <v-row class="ma-0">
-                <v-col cols="auto" class="pl-0 pr-0">
-                  <v-icon :icon="method" size="52"></v-icon>
-                </v-col>
-                <v-col>
-                  <v-chip color="green" class="rounded text-h5"
-                    >{{ mileageStore.totalKmByUser }} KM</v-chip
-                  >
-                  <v-card-subtitle>TOTAL</v-card-subtitle>
-                </v-col>
+                <v-row align="center" class="my-1">
+                  <v-icon :class="['mdi', 'ml-2', method]" size="52" color="#2c3d4f" />
```

Comment:
Is this colour for the method icon? I can't find it used anywhere else and the method icons need to be consistent across pages

Reply:
They are the same colour for some reason only that icon wants to be black so i had to manually set the colour XD